src/Surfnet/ServiceProviderDashboard/Application/Service/EntityService.php

```diff
@@ -161,15 +161,15 @@ public function getEntityListForService(Service $service)
 
         $testEntities = $this->findPublishedTestEntitiesByTeamName(
             $service->getTeamName(),
-            $this->testManageConfig->getPublicationStatus()->getCreateStatus()
```

Comment:
It would be cleaner to use a more explicit function here, instead of casting to string.
Even a (non magic) `toString` method would be beneficial. (Though i think within the domain `status(): string` may be appropriate.

This may help prevent bugs in case `getPublicationStatus()` may become nullable for example. In that case casting it to string will still 'work' but cause harder to debug issues. But if an explicit method call is done, then it will show exactly where the error is happening.

Reply:
Yes, a good suggestion. I'm replacing the __toString with a regular getter.